cDirectX.cpp

```diff
@@ -533,6 +533,19 @@ int Render()
 					info.yaw = ActorYaw;
 					ActorArray.push_back(info);
 				}
+				///
+				/// SHIPWRECKS
+				///
+				else if (name.find("BP_Shipwreck_01_a_NetProxy_C") != std::string::npos || name.find("BP_Shipwreck_") != std::string::npos)
+				{
+					info.id = ActorID;
+					info.name = "SHIPWRECK";
+					info.type = animalcrate;
```

Comment:
Type animal crate?

Reply:
I didn't really know what type to put it under lol, so I just went with animal crate like I did with Sunken Curse statues lol